pkg/server/server_test.go

```diff
@@ -34,9 +34,17 @@ import (
 	"github.com/tbd54566975/ssi-service/pkg/storage"
 )
 
+var storageProvider storage.Type
+
 func TestMain(t *testing.M) {
 	testutil.EnableSchemaCaching()
-	os.Exit(t.Run())
+	providersToTest := []storage.Type{storage.Bolt, storage.InMemory}
```

Comment:
A bit gross, but very effective way of making sure all the tests in the package pass with the new implementation.